src/main/java/net/dv8tion/jda/api/utils/messages/MessageEditData.java

```diff
@@ -353,6 +359,6 @@ public synchronized void close()
 
     protected boolean isSet(int flag)
     {
-        return (set & flag) != 0;
+        return isReplace || (set & flag) != 0;
```

Comment:
So this might have unintended side effects.
```java
MessageEditBuilder builder1 = new MessageEditBuilder();
builder1.setContent("A cool thing!");

MessageEditBuilder builder2 = new MessageEditBuilder();
builder2.setAttachments(FileUpload.fromData(new File("./test.txt")));
builder2.setReplace(true);

builder1.applyData(builder2.build());

System.out.println("content: " + builder1.getContent());
System.out.println("files: " + builder1.getAttachments().size());
```

I'd expect the `builder1.getContent()` here to be `"A cool thing!"` because I didn't actually mutate content on `builder2`, but if I'm understanding this correctly, it would actually be `null` or `""` because of the replace?

Reply:
Yes, that's correct, and that does make sense to me. Replace always implies the entire state being replaced.

Reply:
> Replace always implies the entire state being replaced.

Works for me